.appveyor.yml

```diff
@@ -1,26 +1,34 @@
 cache:
-  - c:\tools\vcpkg\installed\
+  - C:\Tools\vcpkg\installed\
+  - C:\Tools\ninja\ninja.exe
 
 configuration: Release
 
 environment:
   VCPKG_DIR: C:\Tools\vcpkg
+  NINJA_DIR: C:\Tools\ninja
   matrix:
-    - generator: "Visual Studio 14 2015"
-      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2015"
-      Platform: x86
-    - generator: "Visual Studio 14 2015 Win64"
-      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2015"
-      Platform: x64
-    # #Uncoment this block once MSVC2017 is patched.
-    # - generator: "Visual Studio 15 2017"
-    #   APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
-    #   Platform: x86
-    # - generator: "Visual Studio 15 2017 Win64"
-    #   APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
-    #   Platform: x64
+    - APPVEYOR_BUILD_WORKER_IMAGE: 'Visual Studio 2015'
+      PLATFORM: x86
+      GENERATOR: 'Ninja'
```

Comment:
If `GENERATOR: 'Ninja'` is valid for all jobs, you might was well simply add it as a global env, just like `VCPKG_DIR` and `NINJA_DIR`.